onnx/test/helper_test.py

```diff
@@ -724,6 +724,26 @@ def test_make_tensor_raw(tensor_dtype: int) -> None:
     np.testing.assert_equal(np_array, numpy_helper.to_array(tensor))
 
 
+# TODO (#4554): remove this test after the deprecation period
+# Test these new functions should not raise any depreaction warnings
+class TestHelperMappingFunctions(unittest.TestCase):
+    @pytest.mark.filterwarnings("error::DeprecationWarning")
+    def test_tensor_dtype_to_np_dtype_not_throw_warning(self) -> None:
+        _ = helper.tensor_dtype_to_np_dtype(TensorProto.FLOAT)
+
+    @pytest.mark.filterwarnings("error::DeprecationWarning")
```

Comment:
I wonder if it is worth checking what happens in case of bfloat16.

Reply:
Tests for bfloat16 looks useful since the mapping for bfloat16 is quite confusing. Adding these tests can prevent future regression. Just added them. PTAL. Thanks!